src/components/atoms/BaseTextArea.atom.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <textarea
+    :id="id"
+    :name="name"
+    :cols="cols"
+    :placeholder="placeholder"
+    :rows="rows"
+    @blur="handleOnBlur"
+    class="default-textarea"
+    :value="modelValue"
+    @input="emit('update:modelValue', $event.target.value)"
+    :disabled="isDisabled"
+    :required="isRequired"
+  ></textarea>
+</template>
+
+<script lang="ts">
+import { defineComponent, SetupContext } from 'vue'
+
+interface Props {
+  id: string
```

Comment:
This is causing a TS error on the `setup` function.

Try this instead: 
```suggestion
id?: string;
```